src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -65,7 +65,8 @@
     "SqlServer": {
         "Initialize": "true",
         "AllowDatabaseCreation": "true",
-        "ConnectionString": "server=(local);Initial Catalog=Dicom;Integrated Security=true",
+        "ConnectionString": "Server=tcp:alimanagedidentitytest.database.windows.net,1433;Database=Dicom;",
```

Comment:
should the connectionstring be changed?

Reply:
Thanks for pointing this out. No, it should not be changed by default. I will undo this change and submit a new iteration.

Reply:
Fixed.